server/src/main/java/org/elasticsearch/cluster/routing/allocation/DiskThresholdSettings.java

```diff
@@ -39,6 +39,20 @@ public class DiskThresholdSettings {
         Setting.Property.Dynamic,
         Setting.Property.NodeScope
     );
+    public static final Setting<ByteSizeValue> CLUSTER_ROUTING_ALLOCATION_LOW_DISK_MAX_HEADROOM_SETTING = new Setting<>(
+        "cluster.routing.allocation.disk.watermark.low.max_headroom",
```

Comment:
Maybe one for @henningandersen: why are we calling this `max_headroom` and not `min_headroom`? In the rising water metaphor I understand "headroom" to be the space between the water level and the ceiling, and these settings are putting a lower bound on the size of this space.

Reply:
Scrap this too, it makes sense now.